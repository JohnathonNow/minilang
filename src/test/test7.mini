var F1 := fun(A, B, C) do
	print('A = {A}, B = {B}, C = {C}\n')
end

F1(1, 2, 3)

var F2 := F1 !! [10]

F2(1, 2, 3)

var F3 := F2 !! [20]

F3(1, 2, 3)

var G1 := print !! [10]

G1(1, 2, 3)