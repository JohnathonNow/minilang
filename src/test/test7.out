A = 1, B = 2, C = 3
A = 10, B = 1, C = 2
A = 10, B = 20, C = 1
10123